contracts/Removal.sol

```diff
@@ -260,7 +259,7 @@ contract Removal is
     uint256,
     bytes memory
   ) public pure override {
-    revert("Removal: ERC 1155 mint disabled");
+    revert MintDisabled();
```

Comment:
I would rather just change how this works. We should:
- stop inheriting `ERC1155PresetMinterPauserUpgradeable`
- change the `mint` to allow minting `n` removals for one supplier
- change `mintBatch` to allow minting `n` removals for `m` suppliers
- perhaps this needs to be further extended one more dimension for projects?

Reply:
So I stopped inheriting `ERC1155PresetMinterPauserUpgradeable` and added in the infrastructure needed otherwise (like inheriting `ERC1155Burnable`, initializing some roles, calling initializers, etc., got rid of the `mint` overrride.  I haven't made any changes to `mintBatch` yet though.  It does mint `n` removals for one supplier (without validation that they all actually belong to that supplier with encoded address). I'm not sure how we would set things up to mint `n` removals for `m` suppliers so I'm going to leave that to a future PR